languages/c/reference/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,57 @@
-# How to implement an C concept exercise
+# How to Implement a C Concept Exercise
 
-TODO: describe how to implement a concept exercise for the C track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes how to implement a concept exercise for the C track. As this document is generic, the following placeholders are used:
```

Comment:
Could you include the changes that were introduced in: 

- https://github.com/exercism/v3/pull/1170
- https://github.com/exercism/v3/pull/1149